.ci/pipelines/acceptance.Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+node('docker') {
+    String DOCKER_IMAGE = "golang:1.15-stretch"
+    String APP_PATH = "/go/src/github.com/elastic/terraform-provider-ec"
+
+    stage('Checkout from GitHub') {
+	    checkout scm
+    }
+    withCredentials([
+        string(credentialsId: 'vault-addr', variable: 'VAULT_ADDR'),
+        string(credentialsId: 'vault-secret-id', variable: 'VAULT_SECRET_ID'),
+        string(credentialsId: 'vault-role-id', variable: 'VAULT_ROLE_ID')
+    ]) {
+        stage("Get EC_API_KEY from vault") {
+            withEnv(["VAULT_SECRET_ID=${VAULT_SECRET_ID}", "VAULT_ROLE_ID=${VAULT_ROLE_ID}", "VAULT_ADDR=${VAULT_ADDR}"]) {
+                sh 'make -C .ci .apikey'
+            }
+        }
+    }
+    docker.image("${DOCKER_IMAGE}").inside("-u root:root -v ${pwd()}:${APP_PATH} -w ${APP_PATH}") {
+        try {
+            stage("Download dependencies") {
+                sh 'make vendor'
+            }
+            stage("Run acceptance tests") {
+                sh 'make testacc-ci'
+            }
+        } catch (Exception err) {
+            throw err
```

Comment:
I don't think you don't actually need this since you're not doing anything with the exception, Groovy/Java don't require a `catch` block

Reply:
from the top of my head, I remember this being necessary to be able to catch the errors, since they're thrown within Docker and otherwise the pipeline returned with success even when there was an error, but maybe that's from a while ago?

Reply:
Hmm I'm not sure actually. That sounds like a very "Jenkins" thing that could well be true, it just stood out cause it reads funny. Not going to cause a problem if you leave it there though 🙂

Reply:
yeah, I think I might just leave it there for now, and venture into removing it later since I'm sure this works 😂